src/theme/components/link.ts

```diff
@@ -0,0 +1,45 @@
+import { ComponentMultiStyleConfig, SystemStyleObject } from '@chakra-ui/react';
+
+import { fontSizes, lineHeights, space } from '../shared';
+
+const linkBaseStyle: SystemStyleObject = {
+  lineHeight: lineHeights.display,
+  fontSize: fontSizes.base,
+};
+
+const enabledBaseStyle: SystemStyleObject = {
+  color: 'blue.700',
+  _hover: {
+    cursor: 'pointer',
+    textDecoration: 'underline',
+  },
+  _active: {
+    textDecoration: 'none',
+    color: 'gray.900',
+  },
+};
+
+const disabledBaseStyle = {
+  color: 'gray.300',
+  _hover: {
+    cursor: 'default',
+    textDecoration: 'none',
+  },
+};
+
+const iconBaseStyle = {
+  width: space['2xl'],
+  height: space['2xl'],
+};
+
+const Link: ComponentMultiStyleConfig = {
+  parts: ['leftIcon', 'rightIcon', 'link', 'disabled'],
```

Comment:
- `Link` or `a` itself doesn't have `disabled` state like button, therefore, the separate style for `disabled` state is necessary